api/src/main/java/org/openmrs/Person.java

```diff
@@ -59,6 +62,9 @@
 	@Field
 	private String gender;
 	
+
+	@Field(analyze = Analyze.YES)
+	@DateBridge(encoding = EncodingType.STRING, resolution = Resolution.YEAR)
```

Comment:
As a general rule, we'd actually want to store the resolution of the birth date as a DAY rather than a YEAR (even for the purposes of getSimilarPeople(), the resolution isn't quite a year, but a fuzzy "3 year period either side of this given year).

Reply:
I can see we do explicitly need a year. If we cannot construct a year query in Lucene, I think we should add a new calculated Lucene-only field of birthyear or something.

Reply:
Alright. Then let me work on adding a Lucene only field. I will first do a small research if there is way within Lucene

Reply:
@ibacher adding a year field for Lucene sounds reasonable. I will do it that way if you think it is finde

Reply:
@ibacher I can set the resolution to DAY and will not end anything on the result.